speed-dreams/source-2.2.3/src/libs/tgfclient/guimenu.cpp

```diff
@@ -217,8 +217,17 @@ gfuiMenuGetBoolean(const char* pszValue, bool bDefault)
     return bDefault;
 }
 
-static bool
-getControlBoolean(void* hparm, const char* pszPath, const char* pszFieldName, bool bDefault)
+// SIMULATED DRIVING ASSISTANCE
+/// @brief         Converts a boolean into a menu bool str 'yes' or 'no'
+/// @param p_bool  The boolean to base the conversion on
+/// @return        The resulting string 'yes' or 'no'
+const char* GfuiMenuBoolToStr(bool p_bool)
+{
+    return p_bool ? GFMNU_VAL_YES : GFMNU_VAL_NO;
+}
+
+bool
+GfuiMenuControlGetBoolean(void* hparm, const char* pszPath, const char* pszFieldName, bool bDefault)
```

Comment:
Now that you've changed the name of the function, the rest should change too, so:
1. add documentation
2. add SIMULATED DRIVING ASSISTANCE CHANGE: Renamed function
3. move bool
4. add p_